db/migrate/20160901010430_change_column_type.rb

```diff
@@ -1,13 +0,0 @@
-class ChangeColumnType < ActiveRecord::Migration
```

Comment:
@mailethiesen it does look like we have two of the same migration in here.  I think @kpat108 is right in removing this.  Can you confirm?